package.json

```diff
@@ -23,47 +24,33 @@
+    "build:only": "tsc && cp package.json build && rollup -c",
+    "postbuild": "uglifyjs build/vega-lite.js -cm --source-map content=build/vega-lite.js.map,filename=build/vega-lite.min.js.map -o build/vega-lite.min.js && npm run schema",
     "build:examples": "npm run data && TZ=America/Los_Angeles scripts/build-examples.sh",
     "build:examples-full": "TZ=America/Los_Angeles scripts/build-examples.sh 1",
     "build:example": "TZ=America/Los_Angeles scripts/build-example.sh",
     "build:toc": "bundle exec jekyll build -q && scripts/generate-toc",
-    "build:site": "browserify site/static/main.ts -p [tsify -p site] -d | exorcist build/site/main.js.map > build/site/main.js",
+    "build:site": "tsc && webpack --config site/webpack.config.js",
     "build:versions": "scripts/update-version.sh",
-    "build:test-gallery": "browserify test-gallery/main.ts -p [tsify -p test-gallery] -d > build/test-gallery/main.js",
     "check:examples": "scripts/check-examples.sh",
     "check:schema": "scripts/check-schema.sh",
-    "clean": "rm -rf build && rm -f vega-lite.* & find -E src test site examples -regex '.*\\.(js|js.map|d.ts)' -delete && rm -f examples/compiled/*.png && find site/examples ! -name 'index.md' -type f -exec rm -f {} +",
+    "clean": "rm -rf build && rm -f examples/compiled/*.png && find site/examples ! -name 'index.md' -type f -delete",
     "data": "rsync -r node_modules/vega-datasets/data/* data",
     "deploy": "scripts/deploy.sh",
     "deploy:gh": "scripts/deploy-gh.sh",
     "deploy:schema": "scripts/deploy-schema.sh",
-    "prestart": "npm run data && npm run build && scripts/index-examples",
-    "start": "nodemon -x 'npm run build:test-gallery' & browser-sync start --server --files 'build/test-gallery/main.js' --index 'test-gallery/index.html'",
-    "poststart": "rm examples/all-examples.json",
     "preschema": "npm run prebuild",
-    "schema": "ts-json-schema-generator --path tsconfig.json --type TopLevelSpec > build/vega-lite-schema.json && npm run renameschema && cp build/vega-lite-schema.json _data/",
+    "schema": "ts-json-schema-generator --path src/index.ts --type TopLevelSpec > build/vega-lite-schema.json && npm run renameschema && cp build/vega-lite-schema.json _data/",
     "renameschema": "scripts/rename-schema.sh",
     "presite": "npm run prebuild && npm run data && npm run build:site && npm run build:toc && npm run build:versions && scripts/create-example-pages",
     "site": "bundle exec jekyll serve --incremental",
     "lint": "tslint -p .",
-    "test": "npm run build:only && npm run test:only && npm run test:runtime && npm run lint",
-    "posttest": "npm run schema && npm run data && npm run mocha:examples",
-    "test:nocompile": "npm run test:only && npm run test:runtime && npm run lint && npm run mocha:examples",
-    "test:only": "nyc --reporter=html --reporter=text-summary npm run mocha:test",
+    "test": "npm run lint && npm run schema && jest && npm run test:runtime",
```

Comment:
Can we run 

1) unit test (without example schema test)
2) lint
3) schema
4) example schema test
5) runtime test

This would be the most optimal for workflow as we normally want to test logic first before linting.

Reply:
Sure. I will change it. The cool thing about running all tests in one step is that they can run in parallel.